cards/cards.css

```diff
@@ -0,0 +1,124 @@
+@import "../for-import/reset.css"; 
+@import "../for-import/colors.css";
+@import "../for-import/temp-buttons.css";
+@import "../for-import/typography.css";
+
+/* style for icon on top of image */
+.card-top {
+    position: relative;
+}
+.icon-on-image {
+    position: absolute;
+    top: 0.5rem;
+    left: 1rem; 
+   color: var(--accent-color) ;
+    font-size: 1.5rem;
+}
+
+.icon-on-image:hover {
+  color: var(--default-color);    
+}
+
+
+.dismiss-on-card {
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem; 
+    font-size: 1.3rem;
+    color: var(--primary-color);
+}
+
+.text-on-image {
+    position: absolute;
+    top: 0.5rem;
+    right: 50%; 
+    color: var(--primary-color);
+    font-weight: 700;   
+}
+
+.heading-text-on-image {
+    margin-top: 2rem; 
+}
+
+/* cards style */
+
+.cards {    
+    width: 21.8rem; /* 350px */
+    height: 30rem;   
+    padding: 0.7rem;
+    
+    border-radius: 5px;
+    text-align: center;
+}
+
+.card-shadow{
+    box-shadow: 0 4px 8px 0 var(--shadow-color-light);    
+}
+
+.card-shadow:hover {
+    transition: 0.3s;
+    box-shadow: 2px 4px 8px 2px  var(--shadow-color-dark);
+}
+
+.img-hero {
+    width: 100%;
+    height: 50%;
+    border-radius: 5px;
+}
+
```

Comment:
Hey Ruchi, cards look great! Just one suggestion, add `cursor: pointer ` to all the clickable elements like buttons and links. Also, Vertical, Horizontal and Text only cards have no hover effect. Are they supposed to be that way in your design or did you forget to add them? Great work on the design otherwise!!

Reply:
Thanks Raksha, done now :)

Reply:
Looks good Ruchi!